### PR TITLE
Create Parser for iosxe/show_cts_role-based_permissions

### LINKED ISSUE #25: 14 - show cts role-based permissions
```
gw1>show cts role-based permissions 
RBACL Monitor All for Dynamic Policies : FALSE
RBACL Monitor All for Configured Policies : FALSE

```

```
gw2>show cts role-based permissions 
IPv4 Role-based permissions default:
        Permit IP-00
IPv4 Role-based permissions from group 42:Untrusted to group Unknown:
        Deny IP-00
IPv4 Role-based permissions from group 199:FAILED_AUTH to group Unknown:
        Deny IP-00
IPv4 Role-based permissions from group 22:Untrusted to group 22:Untrusted:
        Deny IP-00
IPv4 Role-based permissions from group 100:Tech to group 6:Trusted:
        NFS-08
        ACCESS-01
        Deny IP-00
IPv4 Role-based permissions from group 101:Mars to group 5:Trusted:
        Deny IP-00
IPv4 Role-based permissions from group 111:Technicolor to group 9:Printers:
        PRINTING-10
        GENERIC-14
        Deny IP-00
IPv4 Role-based permissions from group 102:Moon to group 6:Printers:
        ICMP-01
        PRINTING-10
        Deny IP-00
IPv4 Role-based permissions from group 199:FAILED_AUTH to group 9999:QUARANTINE_REDIRECT_PORTAL:
        Deny IP-00
RBACL Monitor All for Dynamic Policies : FALSE
RBACL Monitor All for Configured Policies : FALSE

```
```
gw3>show cts role-based permissions 
IPv4 Role-based permissions default:
        Permit IP-00
IPv4 Role-based permissions from group 12:Untrusted to group Unknown:
        Deny IP-00
IPv4 Role-based permissions from group 17:Web to group Unknown:
        Deny IP-00
IPv4 Role-based permissions from group 199:FAILED_AUTH to group Unknown:
        Deny IP-00
IPv4 Role-based permissions from group 5016:Quarantine to group Unknown:
        Deny IP-00
RBACL Monitor All for Dynamic Policies : FALSE
RBACL Monitor All for Configured Policies : FALSE
```

### PR DESCRIPTION
**Pull request by reustaquio22**

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
Same comment as: https://github.com/networktocode/genieparser/pull/34#discussion_r468702223

File: changelog/undistributed.rst [hunk 1]
```diff
@@ -14,8 +14,10 @@
         * show stackwise-virtual
     * Added ShowCtsPacs
         * show cts pacs
-    * Added ShowClassMap
-        * show class-map
+    * Added ShowCtsRoleBasedPermissions
```
Comment (itdependsnetworks) [inline changelog/undistributed.rst]: duplicate

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py [hunk 2]
```diff
@@ -51,6 +52,153 @@ def test_show_cts_pacs_empty(self):
...
+                "action_policy": "Deny",
+                "action_policy_group": "IP-00"
+            },
+            "monitor_dynamic": False,
+            "monitor_configured": False
+        }
+    }
+
+    golden_output1 = {'execute.return_value': '''
+    IPv4 Role-based permissions default:
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/tests/test_show_cts.py]: Can you fix the spacing on this, every line would have 4 spaces with this format.

File: changelog/undistributed.rst (hunk 1 above)
Comment (reustaquio22) [reply changelog/undistributed.rst]: This is resolved

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py (hunk 2 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/tests/test_show_cts.py]: Spacing is fixed

File: src/genie/libs/parser/iosxe/show_cts.py [hunk 3]
```diff
@@ -149,3 +149,166 @@ def filter_lines(raw_output, remove_lines):
...
+        # Index value for each policy which will increment as it matches a new policy
+        policy_index = 1
+        # Index value for each policy group which will increment as it matches a new policy group
+        policy_group_index = 1
+        # Used to populate data for each policy and the policy index will be used as the key.
+        policy_data = {}
+
+        for line in out:
+            # IPv4 Role-based permissions default:
+            if rb_default_capture.match(line):
```
Comment (reustaquio22) [inline src/genie/libs/parser/iosxe/show_cts.py]: If statements are updated

**Comment by itdependsnetworks** [comment]
Please submit pr

**Comment by itdependsnetworks** [comment]
Has there been a PR to Cisco for this?

**Comment by reustaquio22** [comment]
[Cisco PR#219](https://github.com/CiscoTestAutomation/genieparser/pull/219)

**Pull request closed without merging**